tutorials/clustering.md

```diff
@@ -64,14 +146,35 @@ using our informational views:
 SELECT * FROM timescaledb_information.data_node;
 ```
 
-Now that we've created a database and added a couple data nodes,
-let's go ahead and create a distributed hypertable. With the following
-commands, we'll create a distributed hypertable for collecting
-temperature readings from sensors:
+### Create a new role
 
+Now that we've created a database and added a couple data nodes, let's go ahead
+and create a new user role we can use for our distributed database. We'll use
+Timescale's `distributed_exec` function to perform this action on the data
+nodes:
 ```sql
-CREATE TABLE conditions (time timestamptz NOT NULL, device integer, temp float);
+CREATE ROLE testuser WITH LOGIN PASSWORD 'testpass';
+SELECT * FROM distributed_exec($$ CREATE ROLE testuser WITH LOGIN PASSWORD 'testpass' $$);
```

Comment:
Even though this is possible, isn't this a security issue? The connection is not secure, is it? No need to remove the command, but might be a good idea to highlight that this can be insecure.

Reply:
Depends whether postgres is using SSL.  I added a comment to the following paragraph.